src/Admin/CategoryAdmin.php

```diff
@@ -30,6 +32,23 @@ class CategoryAdmin extends ContextAwareAdmin
         'cascade_validation' => true,
     ];
 
+    /**
+     * @var CategoryManagerInterface
+     */
+    protected $categoryManager;
+
+    public function __construct(
+        $code,
+        $class,
+        $baseControllerName,
+        ContextManagerInterface $contextManager,
+        CategoryManagerInterface $categoryManager
```

Comment:
This is a BC break, if this field is required

Reply:
Maybe make it nullable and set it with container if null. With a next major message to remove that code?

Reply:
@jordisala1991 see my update, you mean like this?

Reply:
Yes, exactly. I am not an expert of this bundle, that's why I will let others approve this, but this should make this BC